package.json

```diff
@@ -8,6 +8,8 @@
   "dependencies": {
     "bootstrap": "^3.3.4",
     "crypto-js": "^3.1.2-5",
+    "es5-shim": "^4.1.7",
```

Comment:
Re: ES6 shim - we should probably use the Babel polyfill, since it only
polyfills the things not already covered by Babel.

http://babeljs.io/docs/usage/polyfill/

Re: ES5 - all of those methods are supported by IE9 and up (
http://kangax.github.io/compat-table/es5/).

On Tue, Jul 7, 2015 at 8:13 AM, Chris Montrois <notifications@github.com>
wrote:

> In package.json
> <https://github.com/getsentry/sentry/pull/1594#discussion_r34051058>:
>
> > @@ -8,6 +8,8 @@
> >    "dependencies": {
> >      "bootstrap": "^3.3.4",
> >      "crypto-js": "^3.1.2-5",
> > +    "es5-shim": "^4.1.7",
>
> The ES6 shim adds objects like Set. The shim for ES5 polyfills methods
> like Array.map and Object.keys.
>
>    - ES5 shim: https://github.com/es-shims/es5-shim
>    - ES6 shim: https://github.com/paulmillr/es6-shim/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getsentry/sentry/pull/1594/files#r34051058>.
>